src/ScheduleService/Controllers/WorkspacesController.cs

```diff
@@ -15,52 +14,52 @@ namespace LT.DigitalOffice.DepartmentService.Controllers;
 [ApiController]
 public class WorkspacesController : ControllerBase
 {
-  [HttpGet("{id}")]
+  [HttpGet("get")]
   public async Task<OperationResultResponse<WorkspaceResponse>> GetAsync(
     [FromServices] IGetWorkspaceCommand command,
-    [FromRoute] Guid id)
+    [FromQuery] Guid id)
   {
     return await command.ExecuteAsync(id);
   }
 
-  [HttpGet]
-  public async Task<FindResultResponse<WorkspaceInfo>> GetAllAsync(
-    [FromServices] IGetWorkspacesCommand command,
-    [FromQuery] GetWorkspacesFilter request)
+  [HttpGet("find")]
```

Comment:
@VLiamin is it ok that we again use find, create, etc in requests?

Reply:
No